packages/plugin-less/src/index.ts

```diff
@@ -173,7 +168,15 @@ export const pluginLess = (
         callback: (rule: RspackChain.Rule, type: 'normal' | 'inline') => void,
       ) => {
         callback(rule, 'normal');
-        callback(inlineRule, 'inline');
+
+        // Rsbuild < 1.3.0 does not have RULE.CSS_INLINE.
+        if (chain.module.rules.has(CHAIN_ID.RULE.CSS_INLINE)) {
```

Comment:
It might be preferable to use `CHAIN_ID.RULE.CSS_INLINE && chain.module.rules.has(CHAIN_ID.RULE.CSS_INLINE)` because, theoretically, there could be a rule named `undefined`.

Reply:
this will be better👍